flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/job/rescaling/RescalingHandlers.java

```diff
@@ -108,21 +90,23 @@ public RescalingStatusHandler(
 		}
 
 		@Override
-		protected AsynchronousJobOperationKey getOperationKey(HandlerRequest<EmptyRequestBody, RescalingStatusMessageParameters> request) {
-			final JobID jobId = request.getPathParameter(JobIDPathParameter.class);
-			final TriggerId triggerId = request.getPathParameter(TriggerIdPathParameter.class);
+		public CompletableFuture<AsynchronousOperationResult<AsynchronousOperationInfo>> handleRequest(@Nonnull final HandlerRequest<EmptyRequestBody, RescalingStatusMessageParameters> request, @Nonnull final RestfulGateway gateway) throws RestHandlerException {
+			throw new RestHandlerException("Rescaling is temporarily disabled. See FLINK-12312.", HttpResponseStatus.SERVICE_UNAVAILABLE);
```

Comment:
Should be `throw featureDisabledException()`

Reply:
fixed